Infrastructure.DataAccess/Mapping/ItSystemUsageOverviewReadModelMap.cs

```diff
@@ -38,6 +39,10 @@ public ItSystemUsageOverviewReadModelMap()
                 .HasMaxLength(ItSystem.MaxNameLength)
                 .HasIndexAnnotation("ItSystemUsageOverviewReadModel_Index_ItSystemParentName", 0);
 
+            Property(x => x.ResponsibleOrganizationUnitName)
```

Comment:
hmm mangler index på Id for det er den vi vil søge på. 

Faktisk skal vi ikke bruge name men kun id fordi vi kommer til at lave en rulleliste med hele organisationstræet til søgning:

![image](https://user-images.githubusercontent.com/13116889/114817180-d6038e80-9db9-11eb-9668-12d3e5c6f4cc.png)


Så

1. Fjern ResponsibleOrganizationUnitName

2. Tilføj index på ResponsibleOrganizationUnitId